base_layer/wallet/src/transaction_service/mod.rs

```diff
@@ -192,7 +192,7 @@ where T: TransactionBackend + 'static
         let base_node_response_stream = self.base_node_response_stream();
         let transaction_cancelled_stream = self.transaction_cancelled_stream();
 
-        let (publisher, _) = broadcast::channel(200);
+        let (publisher, _) = broadcast::channel(10000);
```

Comment:
This is a huge change. Bear in mind that this is impacting directly on the memory footprint of the app. I suspect this lagging is an artifact of having the wallet and base node running on the same Tokio runtime and won't be an issue when we split out the wallet. I wouldn't want to forget about this huge buffer then.

Reply:
I think you are correct in this. I have not seen these subscribers lagging again after I picked it up initially, so I will revert this change.